src/main/java/mate/academy/spring/dao/MovieSessionDao.java

```diff
@@ -9,4 +9,8 @@ public interface MovieSessionDao extends GenericDao<MovieSession> {
     List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
 
     Optional<MovieSession> get(Long id);
+
+    Optional<MovieSession> update(MovieSession movieSession);
```

Comment:
```suggestion
   MovieSession update(MovieSession movieSession);
```
If we update it - it is on DB.